src/main/java/me/shikichee/AnchorApplication.java

```diff
@@ -7,6 +7,11 @@
 public class AnchorApplication {
 
     public static void main(String[] args) {
+        String test = null;
+        if(test!=null){
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.yo1000.com/coding_rules#rule_key=squid%3AS2583)